liferay-frontend/liferay-portal
# LPS-161704 Add GET system relationship endpoint to API Explorer.

Related ticket with the prerequisites -> https://issues.liferay.com/browse/LPS-161704
____
## How to test it
* Deploy the following modules:
  * object-api
  * portal-vulcan-api
  * portal-vulcan-impl
  * object-rest-impl
* Ensure that you have the prerequisites on your Liferay.
* Ensure that you have the feature flag enabled -> `feature.flag.LPS-153324=true`
* Go to API Explorer (http://localhost:8080/o/api)
  * Go to headless-admin-user
  * Go to UserAccount. The last endpoint should be the relationships endpoint.
* Type de Id of a user with some relationships

You should see something like this
![image](https://user-images.githubusercontent.com/17163902/188143282-bcd4c75e-f1dd-4a03-9ddd-3f8935cb3b65.png)
